tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
```

Comment:
Does this miss dependencies in the `package.json` for importing the linting libraries/tools?

Reply:
Could you elaborate on this question a little more? What dependencies are you referring to?

Reply:
Nevermind I think this is okay now